nats/src/lib.rs

```diff
@@ -1,28 +1,28 @@
 #[macro_use]
 extern crate wascc_codec as codec;
 
-mod generated;
+extern crate wasmcloud_actor_core as actorcore;
+extern crate wasmcloud_actor_messaging as messaging;
+
 mod natsprov;
 
-const VERSION: &str = env!("CARGO_PKG_VERSION");
-const REVISION: u32 = 2; // Increment for each crates publish
+pub const VERSION: &str = env!("CARGO_PKG_VERSION");
+pub const REVISION: u32 = 3; // Increment for each crates publish
+pub const CAPABILITY_ID: &str = "wasmcloud:messaging"; // used by the Makefile
```

Comment:
`VERSION` and `REVISION` can be removed here, and `CAPABILITY_ID` can be marked `#[allow(unused)]` to address clippy warning